### PR TITLE
@W-8629051@: Pulled the trigger on PMD language deprecation, and excl…

### PR DESCRIPTION
**Pull request by jfeingold35**
…uded deprecated language JARs from the build process.

### CODE REVIEW

File: src/Constants.ts [hunk 1]
```diff
@@ -51,7 +51,6 @@ export enum LANGUAGE {
 	APEX = 'apex',
 	JAVA = 'java',
```
Comment (jbartolotta-sfdc) [inline src/Constants.ts:52]: Can Java be removed?

File: src/lib/pmd/PmdLanguageManager.ts [hunk 2]
```diff
@@ -1,43 +1,23 @@
...
 import {AsyncCreatable} from '@salesforce/kit';
 import { ENGINE } from '../../Constants';
-import {uxEvents} from '../ScannerEvents';
-
-Messages.importMessagesDirectory(__dirname);
-const eventMessages = Messages.loadMessages("@salesforce/sfdx-scanner", "EventKeyTemplates");
 
 const VALID_LANGUAGES_BY_ALIAS: Map<string, string> = new Map([
 	['apex', 'apex'],
 	['java', 'java'],
```
Comment (jbartolotta-sfdc) [inline src/lib/pmd/PmdLanguageManager.ts:10]: Can java be  removed?

**Review by jbartolotta-sfdc — APPROVED** [review]
Looks good, I asked a couple of questions about removing Java references.

File: src/Constants.ts (hunk 1 above)
Comment (jfeingold35) [reply src/Constants.ts:52]: I guess we could. It would save us 1.14 MB. But do we really want to?

**Pull request merged**